### PR TITLE
Add LowFeeValidation Nomination

### PR DESCRIPTION
**Pull request by bonsfi**
Would you marry me Solana? 50 TKPS 💍

### CODE REVIEW

**Comment by mvines** [comment]
😍 Can you please add the `FullInflationFeaturePair` block as well

**Comment by bonsfi** [comment]
Solana said yes!

File: sdk/src/feature_set.rs [hunk 1]
```diff
@@ -120,6 +120,15 @@ pub mod full_inflation {
             solana_sdk::declare_id!("C89S2MdjXuP6UmgmqKpszoUahfXLd4xVeikP8vJMioNE");
         }
     }
+
```
Comment (mvines) [inline sdk/src/feature_set.rs:123]: ack, whitespace:
```suggestion

```

**Comment by mvines** [comment]
I just pushed a new commit that removes that whitespace, hopefully CI will be happy now

**Comment by mvines** [comment]
@Mergifyio backport v1.5

**Comment by mvines** [comment]
@Mergifyio backport v1.4

**Pull request merged**